creator.html

```diff
@@ -324,18 +318,18 @@ <h5 class="feature-title font-bold mb-1">Frequent Updates</h5>
                             <i class="fa fa-university fa-2x mb-1 deep-purple-text" aria-hidden="true"></i>
                         </div>
                         <div class="col-10 mb-2">
-                            <h5 class="feature-title font-bold mb-1">FREE Forever</h5>
+                            <h5 class="feature-title font-bold mb-1">FREE. Forever</h5>
                             <p class="grey-text mt-2">We decided that Kodular was going to be FREE forever, without any premium plan or compulsory fees for usage.</p>
                         </div>
                     </div>
                 </div>
             </div>
         </section>
-
+        
         <hr class="mb-5">
-
+    
         <section>
-            <h2 class="my-5 h3 text-center">How is it coded?</h2>
+            <h2 class="my-5 h3 text-center">How has been the Creator made with?</h2>
```

Comment:
This part doesn't make sense.

Two suggestions:

```suggestion
            <h2 class="my-5 h3 text-center">How has the Creator been made?</h2>
```

```suggestion
            <h2 class="my-5 h3 text-center">How is the Creator made?</h2>
```